tests/crawler/test_naver_map_crawler.py

```
@@ -1,108 +1,105 @@
+        assert target["1"]["name"] == "Room A"
 
 
 class TestRegionList:
-    """전국 지역 목록 테스트"""
-    
     @pytest.fixture
     def crawler(self):
         return NaverMapCrawler(headless=True)
-    
+
     def test_region_count(self, crawler):
```

Ruff (0.15.1)

[warning] Unused method argument: `crawler`

(ARG002)